worlds/spelunker/docs/en_Spelunker.md

```diff
@@ -0,0 +1,29 @@
+# Spelunker
+
+## Where is the options page?
+
+The [player options page for this game](../player-options) contains all the options you need to configure and export a config file.
+
+## What does randomization do to this game?
+
+All of the collectable items scattered throughout the cave are randomized into the Multiworld pool, with the exception of Energy refills.
+
+## What is the goal of Spelunker when randomized?
+
+The goal is to reach the bottom of the cave and discover the Pyramid of Gold.
+
+## What items and locations get shuffled?
+
+Locations consist of any collectable items aside from Energy refills. Optionally, the player may also randomize the hidden items.

```

Comment:
```suggestion
## What items and locations get shuffled?

The shuffled items and locations are the game's collectible items aside from Energy refills. Optionally, the player may also randomize the hidden items.
```

I'm not super attached to the wording of this suggestion, though you definitely want to change "collect**a**ble" to "collect**i**ble"